curriculum/challenges/english/25-front-end-development/workshop-reusable-profile-card-component/674ef11f75254548672d998c.md

```diff
@@ -0,0 +1,124 @@
+---
+id: 674ef11f75254548672d998c
+title: Step 1
+challengeType: 0
+dashedName: step-1
```

Comment:
`demoType`: `onLoad` is missing here.

Reply:
@moT01 is this necessary?

Reply:
yea, we should add that. It's what makes the final product show up on the first step:

<details><summary>image</summary>

<img width="1399" alt="Screenshot 2024-12-13 at 10 09 26 AM" src="https://github.com/user-attachments/assets/f733f1e6-3043-4f12-b878-802772b43a98" />

</details>

Reply:
Added ✅